packages/cdk/package.json

```diff
@@ -23,12 +23,12 @@
     "ts-jest": "^26.2.0",
     "ts-node": "^10.2.0",
     "typescript": "^3.9.9",
-    "aws-cdk": "^2.2.0"
+    "aws-cdk": "2.38.1"
```

Comment:
nit: would suggest to do something like ^2.38.1 instead of lock a specific version.